website/common/script/libs/inAppRewards.js

```diff
@@ -41,11 +44,13 @@ module.exports = function getPinnedItems (user) {
 
   const pinnedItems = officialPinnedItemsNotUnpinned.concat(user.pinnedItems);
 
-  let items = pinnedItems.map(({type, path}) => {
-    let item = getItemByPathAndType(type, path);
+  let items = pinnedItems
+    .filter(item => Boolean(item)) // filter out nullable entries, which shouldn't be even possible
```

Comment:
aren't this already filtered by the call to `checkPinnedAreasForNullEntries` above?

Reply:
yeah I had this one added  before I added the `checkPinnedAreasForNullEntries` , will be removed